web/islands/PostControls.tsx

```diff
@@ -146,6 +146,26 @@ export function PostControls(props: PostControlsProps) {
       });
   }
 
+  const shareableExternally = navigator.clipboard != null ||
+    navigator.share != null;
+  function onShareExternally() {
+    if (navigator.share) {
+      navigator.share({
+        title: t("post.shareExternally.title", {
+          actor: `${post.actor.name}(${post.actor.handle})`,
+        }),
+        url: localPostUrl,
+      });
+    } else if (navigator.clipboard) {
+      navigator.clipboard.writeText(t("post.shareExternally.fullText", {
+        actor: `${post.actor.name}(${post.actor.handle})`,
+        url: localPostUrl,
+      })).then(() => {
```

Comment:
How about just copying a permalink, without additional text? I think it would be more useful.